native/src/routes/Pois.tsx

```diff
@@ -56,8 +59,9 @@ const Pois = ({
 }: PropsType): ReactElement => {
   const { t } = useTranslation('pois')
   const theme = useTheme()
+  const route = useRoute()
```

Comment:
If you just pass `route` as prop from `PoisContainer`, you don't need this or any type assertions/casts as everything is correctly typed there :)